src/components/SubscriptionsPage.vue

```diff
@@ -53,9 +53,16 @@
             <span v-text="group.groupName !== '' ? group.groupName : $t('video.all')" />
             <div v-if="group.groupName != '' && selectedGroup == group">
                 <i class="i-fa6-solid:pen mx-2" @click="showEditGroupModal = true" />
-                <i class="i-fa6-solid:circle-minus mx-2" @click="deleteGroup(group)" />
+                <i class="i-fa6-solid:circle-minus mx-2" @click="groupToDelete = group.groupName" />
             </div>
+            <ConfirmModal
+                v-if="groupToDelete == group.groupName"
```

Comment:
The modal could be placed outside the for loop with the condition `groupToDelete != null`, that way it's not unnecessarily doing that comparison for each group each time `groupToDelete` changes.

Reply:
@Bnyro done